raiden-dapp/src/store/index.ts

```diff
@@ -126,6 +131,9 @@ const store: StoreOptions<RootState> = {
         value => value.state === ChannelState.open
       );
       return orderBy(openChannels, ['capacity'], ['desc'])[0];
+    },
+    presences: (state: RootState): Presence => {
+      return state.presences;
```

Comment:
Why the trivial getter? `$store.getter.presences` === `$store.state.presences`
Getters are only useful if they do something useful

Reply:
Will remove